demos/colorization_demo/python/colorization_demo.py

```diff
@@ -68,25 +68,24 @@ def build_arg():
 
     log.info('OpenVINO Inference Engine')
     log.info('\tbuild: {}'.format(get_version()))
-    ie = IECore()
+    ie = Core()
 
     log.info('Reading model {}'.format(args.model))
-    load_net = ie.read_network(args.model, args.model.with_suffix(".bin"))
-    load_net.batch_size = 1
+    model = ie.read_model(args.model, args.model.with_suffix(".bin"))
 
-    input_blob = next(iter(load_net.input_info))
-    input_shape = load_net.input_info[input_blob].input_data.shape
+    input_tensor_name = 'data_l'
+    assert input_tensor_name in [input.get_any_name() for input in model.inputs], "Expected input with name 'data_l'"
```

Comment:
not sure that it is our case, but who knows.
a tensor can have several names, `get_any_name` returns any of them (behavior can be different for different compilers), so there is no guarantee that your desired tensor name will be included, although among tensor names it is present. Can we search using input.names() to take into attention all possible names?